pom.xml

```diff
@@ -33,6 +33,13 @@
         <tag>HEAD</tag>
     </scm>
 
+    <repositories>
+        <repository>
+            <id>dl.bintray</id>
+            <url>http://dl.bintray.com/funcheetah8/maven/</url>
```

Comment:
I assume this is temporary and these artifacts will soon be published to Maven Central? We don't want to depend on a third party repository.

Reply:
Yes, I will remove this repository after it is published to Maven Central.

Reply:
@electrum Is it OK to have calcite-core artifact published in https://linkedin.bintray.com/maven/ ?

Reply:
Calcite-core 1.21.0.138 published. Will remove it now.

Reply:
Removed. Both Calcite-core and Coral are now in Maven Central.